CMake/ITKSetStandardCompilerFlags.cmake

```diff
@@ -145,7 +145,10 @@ function(check_compiler_optimization_flags c_optimization_flags_var cxx_optimiza
   if(MSVC)
       set(InstructionSetOptimizationFlags
          # https://docs.microsoft.com/en-us/cpp/build/reference/arch-x64?view=vs-2015
-         /arch:SSE /arch:SSE2 /arch:/AVX ) # /arch:/AVX2 AVX2 circa 2013
+         /arch:SSE /arch:SSE2 /arch:AVX) # AVX2 support circa 2013 hardware
```

Comment:
I think we should update this to remove SSE and SSE2. @hjmjohnson recently did something similar (in VXL?).

Reply:
For WIN 64 it is necessary to remove them. For WIN 32 we can leave them (https://docs.microsoft.com/en-us/cpp/build/reference/arch-x86?view=vs-2015)